pkg/scalers/influxdb_scaler.go

```diff
@@ -103,6 +97,16 @@ func parseInfluxDBMetadata(config *ScalerConfig) (*influxDBMetadata, error) {
 		return nil, fmt.Errorf("no server url given")
 	}
 
+	if val, ok := config.TriggerMetadata["metricName"]; ok {
```

Comment:
Could you please take a look on how Postrgresql scaler is done? We need to mask certain characters in the metricName:
 https://github.com/kedacore/keda/blob/cfebd9e0384c54f23caaeb0074866d5685f0c77a/pkg/scalers/postgresql_scaler.go#L118-L131

For influxdb I think you can use the combination of `prefix (infludb-) + url + organisationName`